src/Microsoft.Health.Fhir.Core/Microsoft.Health.Fhir.Core.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageReference Include="Ensure.That" Version="8.1.1" />
     <PackageReference Include="FluentValidation" Version="8.1.3" />
     <PackageReference Include="MediatR" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="10.0.3" />
```

Comment:
This should not be in Core. You should create a separate assembly and have the Azure dependencies in there.